crates/pglt_statement_splitter/src/parser.rs

```diff
@@ -142,6 +146,21 @@ impl Parser {
         }
     }
 
+    /// Look ahead to the next relevant token *after the next token*
```

Comment:
Der Kommentar ist n bisschen confusing, es ist zwar under the hood `next_pos + 1` aber für clients der API im Prinzip `current + 1`, oder? So liest es sich unten jedenfalls bei dem Ordinality check

Reply:
yeah it's hard to grasp because if you call peek(), the look ahead will basically return the next after peek 😅 what would you suggest as a comment?

Reply:
vllt. einfach `/// Look ahead to the next relevant token`, fühlt sich für mich verständlich an…
`.peek()` könnte auch `current()` sein glaube ich?